darts/utils/timeseries_generation.py

```diff
@@ -256,6 +314,14 @@ def datetime_attribute_timeseries(time_index: Union[pd.DatetimeIndex, TimeSeries
     one_hot
         Boolean value indicating whether to add the specified attribute as a one hot encoding
         (results in more columns).
+    cyclic
```

Comment:
Both the one_hot/cyclic pair and the unit/add_length pair are mutually exclusive, i.e. not both of them can be set, and you correctly check for this. Could you also incorporate this requirement into the docstring?